extensions/2.0/Vendor/GODOT_single_root/GODOT_single_root.md

```diff
@@ -0,0 +1,83 @@
+# GODOT_single_root
+
+## Contributors
+
+* Aaron Franke, The Mirror Megaverse Inc.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension imposes constraints on glTF scenes to ensure they have a single root node.
+
+Many games engines import glTF scenes as objects with a single root node. Unity has prefabs with a single root GameObject, and Godot has scenes with a single root node. Since glTF allows defining multiple root nodes, engines will insert the glTF root nodes as children of the "real" root node, which makes it difficult for a glTF file to define the properties of the "real" root node. This is important for things like physics bodies and character controllers, which are defined on the root node, so that all child nodes are moved with the body. Aside from physics, a single root node would avoid an extra node in the tree when round-tripping between glTF and these engines.
```

Comment:
> which makes it difficult for a glTF file to define the properties of the "real" root node

What are these properties and why cannot scene objects be used for defining them?

Reply:
Properties defined in glTF nodes, especially extensions. I don't think it makes sense to duplicate all of the node logic to apply to scenes, both import and export. You wouldn't add a `"mesh"` to a scene, right?